src/schema/v2/filterArtworksConnection.ts

```diff
@@ -453,27 +466,28 @@ const convertFilterArgs = ({
     artist_series_ids: artistSeriesIDs,
     at_auction: atAuction,
     attribution_class: attributionClass,
-    sizes: sizes,
     dimension_range: dimensionRange,
     exclude_artwork_ids: excludeArtworkIDs,
     extra_aggregation_gene_ids: extraAggregationGeneIDs,
-    include_artworks_by_followed_artists: includeArtworksByFollowedArtists,
-    include_medium_filter_in_aggregation: includeMediumFilterInAggregation,
-    inquireable_only: inquireableOnly,
     for_sale: forSale,
     gene_id: geneID,
     gene_ids: geneIDs,
+    include_artworks_by_followed_artists: includeArtworksByFollowedArtists,
+    include_medium_filter_in_aggregation: includeMediumFilterInAggregation,
+    inquireable_only: inquireableOnly,
+    keyword_match_exact: keywordMatchExact,
+    location_cities: locationCities,
     major_periods: majorPeriods,
     marketing_collection_id: marketingCollectionID,
     materials_terms: materialsTerms,
+    partner_cities: partnerCities,
     partner_id: partnerID,
     partner_ids: partnerIDs,
-    partner_cities: partnerCities,
     price_range: priceRange,
     sale_id: saleID,
+    sizes: sizes,
     tag_id: tagID,
-    keyword_match_exact: keywordMatchExact,
-    location_cities: locationCities,
+    visibility_level: visibilityLevel,
     ..._options,
```

Comment:
This still needs that `include_unpublished: true` if `published: false` thing irrespective (and this will work for admins in the meantime).

Reply:
Updated here: https://github.com/artsy/metaphysics/pull/6408/commits/c2834d0b653c8835d6244cf21ed97e81473f4cef